ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -29,7 +41,2194 @@ console.assert(getHeadmarkerId);
 const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.DragonsongsRepriseUltimate,
   timelineFile: 'dragonsongs_reprise_ultimate.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      headmarkers: {},
+      phase: 'doorboss',
+    };
+  },
+  triggers: [
+    {
+      id: 'DSR Phase Tracker',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['62D4', '63C8'], capture: true }),
+      run: (data, matches) => {
+        switch (matches.id) {
+          case '62D4':
+            data.phase = 'doorboss';
+            break;
+          case '63C8':
+            data.phase = 'thordan';
+            break;
+        }
+      },
+    },
+    {
+      id: 'DSR Headmarker Tracker',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({}),
+      condition: (data) => data.decOffset === undefined,
+      // Unconditionally set the first headmarker here so that future triggers are conditional.
+      run: (data, matches) => {
+        const firstHeadmarker = firstMarker[data.phase];
+        data.headmarkers[matches.target] = getHeadmarkerId(data, matches, firstHeadmarker);
+      },
+    },
+    {
+      id: 'DSR Holiest of Holy',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '62D4', capture: false }),
+      infoText: (_data, _matches, output) => output.aoe!(),
+      outputStrings: {
+        aoe: Outputs.aoe,
+      },
+    },
+    // 0x62D1 = Holy Shield Bash, single target damage on tethered player
+    // 0x63EB = Holy Shield Bash, line charge aoe at tethered player
+    // Charge applies 0xC5D Down for the Count
+    // Followup is 0x62D2 (single target) and 0x62D3 (aoe) Holy Bladedance tank busters
+    {
+      id: 'DSR Empty Dimension',
```

Comment:
Empty = In + Tank Tether, Full = Out + Tank Tether.

Reply:
Ah, was wondering what the Full one was, classic FF 50/50 it didn't even happen once in the instance I was in 🙃

Reply:
Errr actually on second thought I wonder a little if the first one is always empty and the second always full (or random)? I haven't looked in too much detail.

Reply:
Got Empty on both every single run. That's why I was confused lol.

Reply:
Looking more closely, I think it is always Empty for (1) and then Empty/Full for (2).

Reply:
for my group first was alsways empty and second was random